axelrod/__init__.py

```diff
@@ -3,10 +3,18 @@
 
 # The order of imports matters!
 from axelrod.version import __version__
+from axelrod.action import Action
+from axelrod.random_ import Pdf, RandomGenerator, BulkRandomGenerator
+
+# Initialize module level Random
+# This is initially seeded by the clock / OS entropy pool
+# It is not used if user specifies seeds everywhere and should only be
+# used internally by the library and in certain tests that need to set
+# its seed.
+_module_random = RandomGenerator()
```

Comment:
What's the rational of calling this `_module_random` ?  Would `_random_generator` or `_runtime_rng` be a bit more descriptive? (Working through `test_filtering` and was not immediately sure what `_module_random` was.)

Reply:
I didn't think that hard about the name, we can change it. The idea was that it's a "module level" rng and so reproducibility wouldn't be guaranteed. But I think we could also probably eliminate it altogether, if we're willing to toss a few more warnings / exceptions at the user. The biggest issue in that case is instantiating evolvable players with insufficient parameters (for random initialization).

Reply:
Turns out we can remove the need for a seed from the current filter test and it still works.